spec/features/gherkin_dsl_framework.feature

```diff
@@ -1,90 +0,0 @@
-Feature: Rulezilla Gherkin DSL
-
-  Scenario: Condition: something is something
-    Given the gherkin is:
-      """
-        Feature: Awesomeness Rule
-
-        Scenario: Robocop
-          When the "target" is "Robocop"
-          Then the awesomeness is "very awesome"
-      """
-    When the record has attribute "target" and returns "Robocop"
-    Then the result is "very awesome"
-
-
-  Scenario: Multiple Condition: something is something
-    Given the gherkin is:
-      """
-        Feature: Winner Rule
-
-        Scenario: Robocop vs Ironman
-          When the "target" is "Robocop"
-          And the "opponent" is "Ironman"
-          Then the winner is "Ironman"
-      """
-    When the record has attribute "target" and returns "Robocop"
-    And the record has attribute "opponent" and returns "Ironman"
-    Then the result is "Ironman"
-
-
-  Scenario: Default
-    Given the gherkin is:
-      """
-        Feature: Winner Rule
-
-        Scenario: Default
-          When none of the above
-          Then the winner is "Ironman"
-      """
-    Then the result is "Ironman"
-
-
-  Scenario: 'The :keyword is :value', Keyword mismatch
-    Given the incorrect gherkin is:
```

Comment:
should "spec/features/step_definitions/rulezilla_dsl_framework_steps.rb" also be deleted?

Reply:
Actually, not deleted, but some parts removed in that file... search for "the incorrect gherkin"